frontend/lib/verification/remote_verification.dart

```diff
@@ -0,0 +1,18 @@
+import 'verification_card_details_model.dart';
+import 'verification_error.dart';
+import 'verification_hasher.dart';
+
+Future<void> verifyCardValidWithRemote(
+    VerificationCardDetailsModel model) async {
+  try {
+    await Future.delayed(const Duration(seconds: 2), () => "2");
```

Comment:
What is that line for? Particularly the `=> "2"` part?

Reply:
The delay is for demo purposes, I think. The `=> "2"`, I don't know :D

Reply:
Copied from stackoverflow :P Yeah for demo only